source/embed.py

```diff
@@ -16,40 +16,36 @@
 # The functions can be also imported into another Python code
 
 
-import re
+import argparse
+import logging
 import os
-import tempfile
+import re
 import sys
+import tempfile
 import time
-import argparse
-import numpy as np
-import logging
 from collections import namedtuple
-from subprocess import run
 from pathlib import Path
-from typing import Optional, Tuple, Union
-
-import torch
-import torch.nn as nn
-
+from subprocess import run
+from typing import Optional, Union
 
-from lib.text_processing import Token, BPEfastApply, SPMApply
+assert os.environ.get("LASER"), "Please set the environment variable LASER"
+LASER = os.environ["LASER"]
+sys.path.append(LASER)
 
-from fairseq.models.transformer import (
-    Embedding,
-    TransformerEncoder,
-)
-from fairseq.data.dictionary import Dictionary
-from fairseq.modules import LayerNorm
+import numpy as np
+from lib.text_processing import BPEfastApply, SPMApply, Token
+from laser_encoders.models import SentenceEncoder
```

Comment:
Now that `embed.py` depends on `laser_encoders`, we need to add a command for installation of `laser_encoders` (or at least its dependencies: `sentencepiece and sacremoses`) into the `install_external_tools.sh` script, that is currently recommended in our repo for installing dependencies.